client/src/partials/templates/bhFindDocument.tmpl.html

```diff
@@ -0,0 +1,170 @@
+<div class="patient-document">
+
+    <!-- Patient details informations  -->
+    <div ng-if="$ctrl.enablePatientDetails" class="pd-patient">
+      <b><i class="fa fa-universal-access"></i> {{ $ctrl.session.patient.reference }} / </b>
+      {{ $ctrl.session.patient.first_name + ' ' + $ctrl.session.patient.last_name + ' ' + $ctrl.session.patient.middle_name }}
+    </div>
+
+    <!-- Search Documents  -->
+    <div ng-if="$ctrl.enableSearch" class="pd-search">
+      <div class="input-group">
+        <span class="input-group-addon">
+          <i class="fa fa-search"></i>
+        </span>
+        <input
+          ng-model="$ctrl.searchByName"
+          type="text"
+          placeholder="{{ 'FORM.PLACE_HOLDERS.ENTER_NAME' | translate }}..."
+          class="form-control">
+      </div>
+    </div>
+
+    <!-- Documents  -->
+    <div class="pd-documents">
+
+      <!-- heading  -->
+      <div ng-if="$ctrl.enableOptionBar" class="heading clearfix">
+        <div class="pull-left btn">
+          <i class="fa fa-file"></i> {{ 'PATIENT_DOCUMENT.DOCUMENT_LIST' | translate }}
+          ({{ $ctrl.session.patientDocuments.length }})
+        </div>
+
+        <div class="pull-right btn-group">
+          <button
+            type="button"
+            title="{{ 'PATIENT_DOCUMENT.DISPLAY_THUMBNAIL' | translate }}"
+            class="btn btn-sm btn-primary"
+            ng-click="$ctrl.switchDisplay('thumbnail')">
+            <i class="fa fa-file-image-o"></i>
+          </button>
+
+          <button
+            type="button"
+            title="{{ 'PATIENT_DOCUMENT.DISPLAY_LIST' | translate }}"
+            class="btn btn-sm btn-primary"
+            ng-click="$ctrl.switchDisplay('list')">
+            <i class="fa fa-list"></i>
+          </button>
+
+          <button
+            type="button"
+            title="{{ 'PATIENT_DOCUMENT.ADD' | translate }}"
+            class="btn btn-sm btn-primary"
+            ng-click="$ctrl.addDocument()">
+            <i class="fa fa-plus"></i>
+          </button>
+        </div>
+      </div>
+
+      <!-- list of documents  -->
+      <ng-switch on="$ctrl.session.display">
+        <!-- list dispaly mode  -->
+        <div ng-switch-default class="display-list">
+          <div class="">
+            <ul class="media-list">
+              <li class="document media" ng-click="$ctrl.toggleAction($index)"
+                ng-repeat="doc in $ctrl.session.patientDocuments | filter: $ctrl.searchByName">
+                <a class="pull-left">
```

Comment:
Do you need this "pull-left"?  I think the same thing can be accomplished with bootstrap's `media-left` tag.  [See here for documentation](http://getbootstrap.com/components/#media-list).